src/paulscode/android/mupen64plusae/PlayMenuActivity.java

```diff
@@ -213,6 +210,22 @@ protected void onResume()
         mPrefs.registerOnSharedPreferenceChangeListener( this );
         mMogaController.onResume();
         refreshViews();
+        
+        if( ACTION_RESUME.equals( action ) )
```

Comment:
It's been awhile since I looked at it, and perhaps it's broken, but GameActivity is *intended* to be called from any activity.  Is it possible to just cut/paste/replumb PlayMenuActivity.launchGame into GalleryActivity?